wiki/Beatmap_Spotlights/Chart_Assembly_Team/it.md

```diff
@@ -0,0 +1,25 @@
+---
+tags:
+  - cat
+  - cats
+  - charts assembly team
+---
+
+# Team di montaggio dei grafici
```

Comment:
"Grafici" is the wrong term, but i can't think of another better term for this.

Reply:
Wrong as it leads to think of visual graphs

Reply:
Idk either tbf

Reply:
Let's fall back to the original name until we find a better alternative